scripts/hex_root.gd

```diff
@@ -20,6 +20,14 @@ var file_to_delete = ""
 
 func _ready():
 	randomize()
+
+	# The API used below is only available starting with Godot 3.1
+	var version = Engine.get_version_info()
+	if version.major > 3 || (version.major == 3 && version.minor >= 1):
+		# GLES2 renders a black screen if WorldEnvironment background mode is "Canvas"
+		if OS.get_current_video_driver() == OS.VIDEO_DRIVER_GLES2:
```

Comment:
Note that this API is only available in Godot 3.1 beta 5 and later, I just added it.